.drone.yml

```diff
@@ -13,7 +13,6 @@ steps:
   - npm run lint
 - name: dependency-lint
   image: danlynn/ember-cli:3.28.5-node_16.13
-  failure: ignore
```

Comment:
It seems there are no dependency issues so we can make it more strict so we don't accidentally introduce dependency conflicts.